### ISSUE TITLE
Adapt to new GNOME 42 icon colors

### ISSUE THREAD

**Issue by kwbr** [CleoMenezes/Custom-Adwaita-Folder-Icons#5]
Any plans to modify your icons to the new GNOME 42 Adwaita blue-ish style?

Really like and use your icons a lot.

**Comment by CleoMenezesJr** [comment]
Yes, there are already plans. I'm just waiting for the GNOME team to decide what the final color of the folders will be.

**Comment by nylki** [comment]
Was just upgrading Gnome 42, noticed the color differences and was wondering if my favorite icon set already got updated images I could checkout :)

Did you manually create the icons with your base template or did you use a script to render the png's out?

**Comment by CleoMenezesJr** [comment]
First, thank you very much. It's great to read this sort of thing.
The old project is on another branch called GNOME 3, on this new branch I decided to start from scratch and make a couple of changes to make it easier to contribute.
As I started the project being quite unpretentious, I was just creating the icons and saving them in a repository until I lost the SVG that was based on it.
With the new version I left the base SVG file in the / of the project, now each icon has its own directory with its own SVG file and icons.

As I'm short on time, I'm creating on demand. I see that something is missing or someone tells me, so I create it and upload it to the repository.
I'll be happy if you tell me which ones you're using right now to start making.

**Comment by nylki** [comment]
@CleoMenezesJr Fair enough!

Well, I did use quite a bit, really helped me organize my folders:
`camera_2`, `camera`, `certificate`, `code`, `favorite`, `git`, `graphics`, `html5`, `important`, `office_documents`, `office_drawing`, `school`, `sciene`, `script`, `synchronizing`

Would be superb if I could use those again with the Gnome 42 color scheme if you happen to have time re-creating them! But no worries, I'll survive without them :)

**Comment by DanielSvindseth** [comment]
I just forked this repo and uploaded some updated icons that are missing from the new branch. These icons are, however, just copy-pastes and re-colors of the symbols from the older gnome 3 style icons, made in Krita, and not properly remade ones. I'd be happy to contribute upstream, however I am not sure whether the quality is good enough :)

**Comment by CleoMenezesJr** [comment]
> @CleoMenezesJr Fair enough!
> 
> Well, I did use quite a bit, really helped me organize my folders: `camera_2`, `camera`, `certificate`, `code`, `favorite`, `git`, `graphics`, `html5`, `important`, `office_documents`, `office_drawing`, `school`, `sciene`, `script`, `synchronizing`
> 
> Would be superb if I could use those again with the Gnome 42 color scheme if you happen to have time re-creating them! But no worries, I'll survive without them :)

Hey, I just sent all icons you mentioned, Hope u enjoy it.

**Comment by CleoMenezesJr** [comment]
> I just forked this repo and uploaded some updated icons that are missing from the new branch. These icons are, however, just copy-pastes and re-colors of the symbols from the older gnome 3 style icons, made in Krita, and not properly remade ones. I'd be happy to contribute upstream, however I am not sure whether the quality is good enough :)

I would be happy to receive PR. I'm doing everything from scratch exactly because it was just PNGs, which made any contributions difficult. Today, I provide a preset with all guidelines.

**Comment by nylki** [comment]
> > @CleoMenezesJr Fair enough!
> > Well, I did use quite a bit, really helped me organize my folders: `camera_2`, `camera`, `certificate`, `code`, `favorite`, `git`, `graphics`, `html5`, `important`, `office_documents`, `office_drawing`, `school`, `sciene`, `script`, `synchronizing`
> > Would be superb if I could use those again with the Gnome 42 color scheme if you happen to have time re-creating them! But no worries, I'll survive without them :)
> 
> Hey, I just sent all icons you mentioned, Hope u enjoy it.

Thank you! Just set the new icons. They're looking great. :+1:
